README.md

```diff
@@ -67,6 +67,21 @@ app's Gemfile.
 
 ## Usage
 
+### Configuration
+
+Journaling provides a number of different configuation options that can be set in Ruby using an initializer. Those values are:
+
+#### `Journaled.default_app_name`
+
+  This is described in the proceeding paragraph and is used to specify which app name to use, which corresponds to which Journaled Stream to send events too.
+  Ex: `Journaled.default_app_name = 'my_app'`
+
+#### `Journaled.default_job_priority` (default: 20)
+
+  This can be used to configure what `priority` the Delayed Jobs are enqueued with. This will be applied to all the Journaled::Devivery jobs that are created by this application.
+  Ex: `Journaled.default_job_priority = 14`
```

Comment:
This is no longer a default, it's the only way to set it, right? So I think `job_priority` is a better name if we're taking the stance that journaled jobs should all be delivered with the same priority?